README.adoc

```diff
@@ -104,3 +104,99 @@ include::topics/templates/document-attributes.adoc[]
 . Open the generated html file to ensure everything looks correct.
 
 You have now created the scaffolding for a new guide. When you are ready for some initial review and feedback, file a pull request to `upstream/master`.
+
+### Merging Contributions
+// title OK?
+// following content up for revision.
+// add note on branching according to issue numbers?
+// externalize naming examples to variables?
+
+. Once the changes in a Pull Request have been approved, you can merge them into `upstream/master`.
++
+[IMPORTANT]
+--
+Do not merge your own Pull Requests unless absolutely necessary. You are accountable for any errors, merge conflicts or bugs resulting from incorrect merging. Let another contributor with the appropriate repository access privileges do it.
+--
+
+. Before merging, verify that you have the Contributor's fork of the link:https://github.com/openshiftio/appdev-documentation[upstream repository] repository in your list of remotes.
```

Comment:
Why capital in _Contributor's?_

I would also replace _verify_ with _ensure,_ as that's what most of the CCS team uses.

Should end with a colon.